### PR TITLE
Add reference docs for explain of find and aggregate

### PR DESCRIPTION
**Pull request by jyemin**
JAVA-3909

### CODE REVIEW

**Review by rozza — COMMENTED** [review]
Shouldn't examples be added to reactive & scala as well?

**Comment by jyemin** [comment]
Added reactive and scala docs

**Review by rozza — CHANGES_REQUESTED** [review]

File: docs/reference/content/driver-reactive/tutorials/aggregation.md [hunk 1]
```diff
@@ -93,3 +93,20 @@ collection.aggregate(
       )
 ).subscribe(new PrintDocumentSubscriber());
 ```
+
+### Explain an Aggregation
+
+To [explain]({{< docsref "reference/command/explain/" >}}) an aggregation pipeline, call the
+[`AggregateIterable.explain()`]({{< apiref "mongodb-driver-reactivestreams" "com/mongodb/client/AggregateIterable.html#explain()" >}})
```
[the remark this answers is not in the document]
Comment (jyemin) [reply docs/reference/content/driver-reactive/tutorials/aggregation.md]: Copy pasta.  Done.

File: docs/reference/content/driver-reactive/tutorials/perform-read-operations.md [hunk 2]
```diff
@@ -170,6 +170,20 @@ collection.find(and(gte("stars", 2), lt("stars", 5), eq("categories", "Bakery"))
           .subscribe(new PrintDocumentSubscriber());
 ```
 
+### Explain
+
+To [explain]({{< docsref "reference/command/explain/" >}}) a find operation, call the
+[`FindIterable.explain()`]({{< apiref "mongodb-driver-reactivestreams" "com/mongodb/client/FindIterable.html#explain()" >}})
```
[the remark this answers is not in the document]
Comment (jyemin) [reply docs/reference/content/driver-reactive/tutorials/perform-read-operations.md]: Done

File: docs/reference/content/driver-scala/tutorials/aggregation.md [hunk 3]
```diff
@@ -86,3 +86,20 @@ collection.aggregate(
   )
 ).printResults()
 ```
+
+### Explain an Aggregation
+
+To [explain]({{< docsref "reference/command/explain/" >}}) an aggregation pipeline, call the
+[`AggregateIterable.explain()`]({{< apiref "mongo-scala-driver" "com/mongodb/client/AggregateIterable.html#explain()" >}})
```
[the remark this answers is not in the document]
Comment (jyemin) [reply docs/reference/content/driver-scala/tutorials/aggregation.md]: Done

File: docs/reference/content/driver-scala/tutorials/perform-read-operations.md [hunk 4]
```diff
@@ -156,6 +156,20 @@ collection.find(and(gte("stars", 2), lt("stars", 5), equal("categories", "Bakery
           .printResults()
 ```
 
+### Explain
+
+To [explain]({{< docsref "reference/command/explain/" >}}) a find operation, call the
+[`FindIterable.explain()`]({{< apiref "mongo-scala-driver" "com/mongodb/client/FindIterable.html#explain()" >}})
```
[the remark this answers is not in the document]
Comment (jyemin) [reply docs/reference/content/driver-scala/tutorials/perform-read-operations.md]: Done

**Review by rozza — APPROVED** [review]
LGTM!

**Pull request merged**